contracts/Commitment/WhitelistedCommitment.sol

```diff
@@ -139,6 +139,7 @@ contract WhitelistedCommitment is AccessRoles, CommitmentBase {
 
     function reclaim(IBasicToken token)
         public
+        only(ROLE_RECLAIMER)
```

Comment:
Isn't this redundant with the check that already happens in `Reclaimable.reclaim` ?

Reply:
Yeah, it is. I am not sure what would be our general strategy for such situations? 

I would argue to leave it here because in future someone can modify this method (call another contract or something) assuming that the only reclaimer could do it but in a reality, everyone can.

Reply:
Agreed. I like the added readability. The extra gas cost is neglible since this is an 'admin' call anyway.

For sake of consistency we should then do the same on `LockedAccount` and `EtherToken`.